### PR TITLE
[50_1] New Entry for Mogan Draw

### PR DESCRIPTION
**Pull request by Oyyko**

### CODE REVIEW

File: xmake.lua [hunk 1]
```diff
@@ -485,6 +485,228 @@ if is_plat("mingw") then
     end
 end
 
+target("draw") do
```
Comment (darcy-shen) [inline xmake.lua:488]: just ignore the code for mingw and macos

it should only be enabled on Linux and wasm

Comment (darcy-shen) [reply xmake.lua:488]: Focusing on wasm and linux, will make the build definition much more simpler.

**Comment by darcy-shen** [comment]
It depends on #914, in general, it is ok. To make it work on WASM, we just need to focus on Linux and wasm, based on the restricted platform fact, we can just remove the mingw and macOS related dead code.

File: xmake.lua (hunk 1 above)
Comment (darcy-shen) [reply xmake.lua:488]: We can use set_enabled here: https://xmake.io/#/manual/project_target?id=targetset_enabled

eg:
```
set_enabled(is_plat("linux" or is_plat("wasm"))
```

**Comment by Oyyko** [comment]
This branch is a little mess... I will close this PR and create a new branch for the new entry of mogan draw.

**Pull request closed without merging**